ISSUER_PROTOCOL.md

```diff
@@ -18,22 +18,29 @@ A Trust Token issuer should have an endpoint at a publicly accessible secure URL
 ```
 Key commitment result
 {
-  "protocol_version": <protocol version, TrustTokenV3PMB or TrustTokenV3VOPRF for this>,
-  "id": <key commitment identifier, as a monotonically increasing integer>
-  "batchsize": <batch size>,
-  "srrkey": <base-64 encoded SRRVerificationKey, in 32-byte RFC8032 encoding>,
-  <keyID>: { "Y": <base64-encoded TrustTokenPublicKey>,
-                     "expiry": <key expiry, encoded as a string representation of
-                                an integer timestamp in microseconds since the Unix
-                                epoch> },
-  <keyID>: { "Y": <base64-encoded public key>,
-                     "expiry": <key expiry, encoded as a string representation of
-                                an integer timestamp in microseconds since the Unix
-                                epoch> }, ...
+  <protocol_version>: {
+    "protocol_version": <protocol version, TrustTokenV3PMB or TrustTokenV3VOPRF for this>,
```

Comment:
nit: should we quote the strings? "TrustTokenV3PMB"

I know it wouldn't be valid JSON to have them unquoted, but I figure we might as well make things easier on the reader

Reply:
Done.